src/com/sun/ts/tests/jaxws/ee/w2j/rpc/literal/sec/secbasic/jakartaEE.webservices.xml

```diff
@@ -17,11 +17,11 @@
 
 -->
 
-<webservices xmlns="http://java.sun.com/xml/ns/javaee"
+<webservices xmlns="https://jakarta.ee/xml/ns/jakartaee"
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              version="1.2"
 	     xmlns:wsdl="http://BasicAuthServiceTestService.org/wsdl"
-             xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/javaee_web_services_1_2.xsd">
+             xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/javaee_web_services_1_2.xsd">
```

Comment:
should be https://jakarta.ee/xml/ns/jakartaee/jakartaee_web_services_2_0.xsd and version 2.0

Reply:
fixed